requirements.txt

```diff
@@ -43,6 +51,8 @@ six==1.16.0
 tomlkit==0.13.0
     # via reuse
 urllib3==2.2.1
```

Comment:
do you need urllib3 if you have requests already?

Reply:
I think urllib3 is lower-level than requests, and it looks like that latter depends on it

Reply:
Yes and I assumed `requests` would pull `urllib3` anyway. I am thinking on the lines of that if we use `urllib3` elsewhere we can just change that code to use `requests`.

Reply:
oh, I think this file is equivalent to Cargo.lock... it shows transitive deps in addition to direct ones